src/aktualizr_lite/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ add_aktualizr_test(NAME lite-helpers SOURCES helpers.cc helpers_test.cc
                    ARGS ${PROJECT_BINARY_DIR}/ostree_repo)
 set_tests_properties(test_lite-helpers PROPERTIES
         ENVIRONMENT LD_PRELOAD=$<TARGET_FILE:t_lite-mock> LABELS "noptest")
-
+else(BUILD_OSTREE)
+	message(WARNING " acktualizer-lite not configured, skipping")
 endif(BUILD_OSTREE)
 
```

Comment:
I'm not sure upstream would want this, but I might be confused about this. So the  point of this is that cmake will warn you if aktualizr-lite isn't going to be built? If so, aktualizr-lite isn't built by default in upstream, so I don't think they are going to want to be warned about that?

Reply:
I dont know...for me, just someone curious about the code, it was a PIA using cmake to build and finding that the the lite binary was not being built with no info about it (had to add  -DBUILD_OSTREE to cmake)...anyway, you can ignore it of course